docs/docs/building-and-running.md

```diff
@@ -141,7 +134,7 @@ helm repo update
 # update magda chart dependencies
 helm dep build deploy/helm/magda
```

Comment:
we probably need to update doc here as users need to build dependencies for `magda-core` as well
@sajidanower23

Reply:
Added